third_party/2and3/atomicwrites/__init__.pyi

```diff
@@ -1,15 +1,22 @@
+import sys
 from typing import Any, AnyStr, Callable, ContextManager, Generic, IO, Optional, Text, Type, Union
 
+if sys.version_info >= (3, 6):
+    from os import PathLike
+    _Path = Union[Text, bytes, PathLike[AnyStr]]
```

Comment:
```suggestion
    _Path = Union[str, bytes, PathLike[Union[str, bytes]]
```

Since we are in a Python 3-only branch, it's clearer to just use `str` instead of `Text`. Also, `AnyStr` is a type variable, so using it once makes no sense. Just use `Union[str, bytes]`.

Reply:
The first change (`Text` -> `str`) makes sense.  The second one does not seem to be this straight forward; your suggestion would lead to an error:

```text
third_party/2and3/atomicwrites/__init__.pyi:6: error: Invalid type argument value for "_PathLike"
```

I suppose I could work around that like this?

```suggestion
    _Path = Union[str, bytes, PathLike[str], PathLike[bytes]]
```

Reply:
Good point, let's try that. If that doesn't work either, just revert to `AnyStr`.

Reply:
Seems to work! I pushed a new version